samples/bot-receive-channel-messages-withRSC/csharp/README.md

```diff
@@ -77,6 +77,7 @@ This feature shown in this sample is currently available in Public Developer Pre
 
 5. This step is specific to Teams.
     - **Edit** the `manifest.json` contained in the  `AppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`) also update the `<<DOMAIN-NAME>>` with the ngrok URL`
+    - **Edit** the `manifest.json` for `<<MANIFEST-ID>>` replace it with any `GUID` using guid generator - [Guid Generator](https://guidgenerator.com/online-guid-generator.aspx).
```

Comment:
Not sure how long the referenced tool would be available. We can keep the existing GUID in manifest.